README.md

```diff
@@ -47,14 +47,10 @@ how to use them safely, and an example swap between Decred and Bitcoin.
 
 Pre-requirements:
 
-  - Go 1.9 or later
-  - [dep](https://github.com/golang/dep)
+  - Go 1.11 or later
 
 ```
-$ cd $GOPATH/src/github.com/decred
-$ git clone https://github.com/decred/atomicswap && cd atomicswap
-$ dep ensure
-$ go install ./cmd/...
+$ go get github.com/decred/atomicswap/cmd/...
```

Comment:
these build instructions won't work. `go get` is unable to build and install full programs when in module mode (instead, it will attempt to add the module(s) to the current module).  `go get` may regain that functionality in 1.12 but for now these instructions will have to specify how to clone the repo, cd into the directory for the tools you wish to build, and then run `go build` or `go install` there.